### PR TITLE
[ZH] Add `Spotlights/2022_Summer`

### PR DESCRIPTION
**Pull request by Kotoki1337**
- Create `zh.md` for `2022_Summer`
- Update index page for all language

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by cl8n** [comment]
was this reviewed off-platform?

**Comment by Kotoki1337** [comment]
> was this reviewed off-platform?

no, sorry, idk when i wrongly checked the check box, this is not reviewed

File: wiki/Beatmap_Spotlights/Seasons/2022_Summer/zh.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+| 课题模式大厅：课题 A | 2022 年 9 月 22 日 - 9 月 29 日 |
+| 课题模式大厅：课题 B | 2022 年 9 月 29 日 - 10 月 6 日 |
+| 课题模式大厅：课题 C | 2022 年 10 月 6 日 - 10 月 13 日 |
+| 赛季结算 | 2022 年 10 月 13 日 |
+
+## 每周多人游戏大厅
+
+目前，加入每周多人游戏大厅需要使用 [osu!(lazer)](https://github.com/ppy/osu/releases)。
+
+1. 赛季每 2 周将开放一个课题模式大厅，每大厅在整个开放期间都可参加。
```
Comment (kj415j45) [inline wiki/Beatmap_Spotlights/Seasons/2022_Summer/zh.md:29]:
```suggestion
1. 赛季开始后每 2 周将开放一个课题模式大厅，每个大厅在其开放期间都可参加。
```

File: wiki/Beatmap_Spotlights/Seasons/2022_Summer/zh.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+   - 在第一轮课题结束后，每周将重新开放一个课题。
```
Comment (kj415j45) [inline wiki/Beatmap_Spotlights/Seasons/2022_Summer/zh.md:30]:
```suggestion
   - 在第一轮课题结束后，频率降低为每 1 周重新开放一个课题。
```

File: wiki/Beatmap_Spotlights/Seasons/2022_Summer/zh.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+2. 每个大厅会提供一个提前准备好的课题。
+   - 每个课题内所包含的 Hard，Insane 和 Expert 难度均匀分布。
+   - 每个课题与每两周分配的字母相对应。每个课题会在下个周期复刻一次。
```
Comment (kj415j45) [inline wiki/Beatmap_Spotlights/Seasons/2022_Summer/zh.md:33]: 读起来很怪

**Review by kj415j45 — CHANGES_REQUESTED** [review]

File: wiki/Beatmap_Spotlights/Seasons/2022_Summer/zh.md (hunk 3 above)
Comment (Kotoki1337) [reply wiki/Beatmap_Spotlights/Seasons/2022_Summer/zh.md:33]:
```suggestion
   - 课题与每周分配的字母相对应。所以每个课题在一个赛季中都有一次复刻机会。
```

Comment (heipizhu4) [reply wiki/Beatmap_Spotlights/Seasons/2022_Summer/zh.md:33]:
```suggestion
   - 每个课题与每两周分配的字母相对应。每个课题会在下个周期复现一次。
```

Comment (Kotoki1337) [reply wiki/Beatmap_Spotlights/Seasons/2022_Summer/zh.md:33]: 主要是很多游戏都在用 `复刻` 这个词，代表某个动作或者活动重新返场进行，`复现` 也不错，我个人更倾向 `复刻`。但这里其实替换的是 `replayed`，和上文的 `opened again` 差不多，我有在想要不要把所有课题返场一周的操作统称为 `复刻`，或者继续和上文一样统一为类似 `reopened` 的译文：
```suggestion
   - 课题与每周分配的字母相对应。所以每个课题在一个赛季中都会重新开放一次。
```

Comment (kj415j45) [reply wiki/Beatmap_Spotlights/Seasons/2022_Summer/zh.md:33]:
```suggestion
   - 每个课题均与一个字母相对应。意味着课题会在赛季内复刻一次。
```

**Review by kj415j45 — APPROVED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**